### DISCUSSION TITLE
posts/attack-and-defense-on-cdn/

### DISCUSSION THREAD

**Discussion by giscus** [yunnysunny/yunnysunny.github.io#27] [Show and tell]
# posts/attack-and-defense-on-cdn/

直到有一天，我突然收到了一条又拍云的报警短信，说是我购买的cdn服务 https 请求数过多。我当时多少有些诧异，心想估计是某个爬虫误伤了我的网站，加之当时工作比较忙，就没来得及顾上。等待下班回家，我迫不及待的打开电脑，进去又拍云管理后台，定睛一看，直接被惊呆了，一天竟然有一百多万次请求，肯定是被攻击了。

https://blog.whyun.com/posts/attack-and-defense-on-cdn/

**Comment by truebigsand** [comment]
捉个typo: `redircts`->`redirects`

**Comment by onelittlechildawa** [comment]
是一组固定的ip为何不直接屏蔽ip或者设置访问限制

**Reply by yunnysunny** [reply]
ip屏蔽完之后，cdn厂商的策略是将请求过来的http返回状态码 403，但是这样依赖依然会有网络流量走到 cdn 节点上，所以流量费还是得收取。

**Reply by onelittlechildawa** [reply]
原来如此，感谢指教。
